doc/source/tune/_tutorials/tune-pytorch-cifar.rst

```diff
@@ -0,0 +1,212 @@
+.. _tune-pytorch-cifar:
+
+How to use Tune with PyTorch
+============================
+
+Hyperparameter tuning can make the difference between an average model and a highly
+accurate one. Often simple things like choosing a different learning rate or changing
+a network layer size can have a dramatic impact on your model performance. Fortunately,
+Tune makes exploring these optimal parameter combinations easy - and works nicely
+together with PyTorch.
```

Comment:
I'm thinking we probably want to include a couple typical things you'll want to do for pytorch:

1. setting more CPUs for dataloaders
2. setting more GPUs for DataParallel 
3. saving optimizer state?

Reply:
1) I explicitly mention resource allocation now (and mention the DataLoaders `num_workers` property)
2) I added and tested (multi) GPU support in the example and a small section in the documentation
3) We now save and restore both the optimizer and model state